ports/apocrunner/Apoc Runner.sh

```diff
@@ -13,45 +13,34 @@ else
 fi
 
 source $controlfolder/control.txt
-export PORT_32BIT="Y"
-
-
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
 get_controls
 
-$ESUDO chmod 666 /dev/tty0
-$ESUDO chmod 666 /dev/tty1
-printf "\033c" > /dev/tty0
-printf "\033c" > /dev/tty1
-
+# Variables
 GAMEDIR="/$directory/ports/apocrunner"
+SPLASHFILE="splash.png"
 
-[ -f "/etc/os-release" ] && source "/etc/os-release"
-
-cd "$GAMEDIR"
+# CD and set permissions
+cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+$ESUDO chmod +x -R $GAMEDIR/*
 
-exec > >(tee "$GAMEDIR/log.txt") 2>&1
+# Exports
+export LD_LIBRARY_PATH="$GAMEDIR/lib:$LD_LIBRARY_PATH"
```

Comment:
```suggestion
export LD_LIBRARY_PATH="$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"
```

Also see my comments on libcrypto / libopenal / libzip :)